plugins/orchestrator-backend/src/service/SonataFlowService.ts

```diff
@@ -315,6 +393,16 @@ export class SonataFlowService {
     return false;
   }
 
+  private extractWorkflowType(workflowDef: WorkflowDefinition): string {
+    if (workflowDef.annotations) {
+      const firstValue: string = workflowDef.annotations[0];
```

Comment:
Why `annotation[0]`?

Reply:
/**
     * List of helpful terms describing the workflows intended purpose, subject areas, or other important qualities
     */
    annotations?: [string, ...string[]];

Reply:
My doubt is: if there are multiple annotations and you grab the first one, are you sure that the selected annotation is the one that represent the WorkflowType?

Reply:
just extract the annotation starting by "workflow-type/"

Reply:
[336de2d](https://github.com/caponetto/backstage-plugins/pull/26/commits/336de2d66715613e8a7ae8699a211b7b9f05ee80)